invenio_app_rdm/records_ui/templates/semantic-ui/invenio_app_rdm/records/detail.html

```diff
@@ -46,12 +46,10 @@
       </div>
     {% endif %}
 
-    {% if permissions is defined and permissions.can_edit and not is_preview %}
-    <div class="ui warning flashed top-attached manage message">
-      <div class="ui container">
-        <div id="recordManagement" data-record='{{ record | tojson }}' data-permissions='{{ permissions | tojson }}'></div>
+    {% if record.parent.communities.default %}
```

Comment:
are you sure about this? if there is a request, the community won't appear here no?
Shouldn't this be:
```suggestion
    {% if community %}
```
?